syntax = "proto3";

message ClientAttributes {

    string user_address = 1;
    string family_name = 5;
}

message ClientInfoSetupTransaction{
    ClientAttributes client_info = 1;
}

message StateUpdateTransaction{
    string action = 1;
    bytes data = 2;
}